### ISSUE TITLE
Require the IIT-CDIP Test Collection

### ISSUE THREAD

**Issue by Coldog2333** [kba/awesome-ocr#121]
Dear kba,

Hello, I am Junfeng, a student from the University of Tokyo.

I am wondering do you have the IIT-CDIP Test Collection Dataset?
If so, could you please share it via google drive for academic usage?

Thank you!

Best regards,
Junfeng

**Comment by kba** [comment]
No, sorry, I don't have that dataset myself. C.f. https://github.com/cneud/ocr-gt/issues/12 there are many interested in it, perhaps someone subscribed to this repo knows how to get it...
